packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -250,6 +250,10 @@ governing permissions and limitations under the License.
   &.is-drop-target {
     @inherit: %drop-target;
   }
+
+  .spectrum-Table-cellWrapper:last-child .spectrum-Table-cell--divider {
```

Comment:
This could probably happen with virtualized scrolling - where the last cell currently in the DOM is not necessary the last cell in the whole table. Not sure we have an example like this at the moment, or if you'd ever see it due to over scanning but it may be better to apply a class with JS to the last column...

Reply:
I checked this locally by adding the `showDivider` to the many column and rows table example and like you said it doesn't really seem to be an issue cuz the "last" column DOM cells are always out of view (cuz over scanning) until you scroll to the actual last column.

@ktabors if you wanna do the JS solution, perhaps you can check `cell.column.nextKey !== null` (last column won't have a next key) or check that `cell.column.key` is equal to the last column key in `collection.columns`. Then based on that, adjust https://github.com/adobe/react-spectrum/blob/main/packages/@react-spectrum/table/src/TableView.tsx#L621 to omit the class if it is the last column